sections/sections-app/src/java/org/sakaiproject/tool/section/jsf/backingbean/EditManagersBean.java

```diff
@@ -73,7 +75,8 @@ public class EditManagersBean extends CourseDependentBean implements Serializabl
 		public int compare(Object o1, Object o2) {
 			ParticipationRecord manager1 = (ParticipationRecord)o1;
 			ParticipationRecord manager2 = (ParticipationRecord)o2;
-			return manager1.getUser().getSortName().compareTo(manager2.getUser().getSortName());
+			Collator collator = Collator.getInstance(new Locale(System.getProperty("user.language"), StringUtils.isNotBlank(System.getProperty("user.region")) ? System.getProperty("user.region") : System.getProperty("user.country")));
```

Comment:
is user.language and user.country set by default?

Reply:
does using the locale that is returned from preferences, make sense?

Reply:
It looks like there's already a few patterns that are in use in sections that should work:

` Locale locale = LocaleUtil.getLocale(FacesContext.getCurrentInstance());`

Or you should be able to use the `ResourceLoader`

```
Locale locale = new ResourceLoader().getLocale();
```

Reply:
It works great with 

```
Collator collator = Collator.getInstance();
```